### PR TITLE
reply to requests!

### PR DESCRIPTION
**Pull request by mixmix**

### CODE REVIEW

**Comment by ameba23** [comment]
great that this is already so far.
i just tried to return some of your shards using this branch.  let me know if you got them.

File: views/others-shards/index.js [hunk 1]
```diff
@@ -1,43 +1,132 @@
...
+    }
   }
 
-  function getShards () {
-    const store = MutantArray([])
+  function getRecords () {
     pull(
-      scuttle.shard.pull.friends({ live: true }),
+      scuttle.shard.pull.fromOthers({ reverse: true, live: false }),
+      pull.paramap((shard, cb) => {
```
Comment (KGibb8) [inline views/others-shards/index.js]: I'm sure you've probably had to explain this before to me, but can you remind me what the difference is between `pull.paramap` and `pull.asyncMap`? I wouldn't know when to apply one and not the other. Noticing the comment `// width 3`, what does this mean too?

**Comment by KGibb8** [comment]
Really nice styling @mixmix. I'll change the other view to match this, then we're well standardised. :+1:

File: views/others-shards/index.js (hunk 1 above)
Comment (mixmix) [reply views/others-shards/index.js]: paramap keeps the results that come back in order even if the asynchronous actions it calls complete at different times

**Comment by mixmix** [comment]
looks like the invites don't have the shard attached ... hmm will go look.
Thanks for trying it out all already, that's awesome!

**Comment by mixmix** [comment]
ahh feck, it's this `server.private.unbox` I mentioned earlier @ameba23

**Comment by mixmix** [comment]
Found the problem...

File: views/others-shards/index.js [hunk 2]
```diff
@@ -1,50 +1,136 @@
...
+    pull(
+      scuttle.shard.pull.fromOthers({ live: true, old: false }),
       pull.filter(m => !m.sync),
-      pull.drain(root => store.insert(root, 0))
+      pull.drain(m => getRecords())
     )
-    return throttle(store, 100)
+
+    // TODO watch for others requests
+    // actually only watch requests...
```
Comment (mixmix) [inline views/others-shards/index.js]: I've made a wekan card for this.
will merge this but come back for this task tmrw

**Comment by ameba23** [comment]
i just made a commit on this branch thinking i was somewhere else... oops.  although i dont think i pushed it (cant see it here).  since this is already merged it should matter anyway

**Pull request merged**